src/client/components/DetailsModal/Details/SwedishCarDetails.tsx

```diff
@@ -19,9 +19,9 @@ export const getSwedishCarValidationSchema = (textKeys: TextKeyMap) => {
         .matches(/^[0-9]{3}[0-9]{2}$/, textKeys.GENERIC_ERROR_INPUT_FORMAT())
         .required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
       city: Yup.string().required(textKeys.GENERIC_ERROR_INPUT_REQUIRED()),
-      registrationNumber: Yup.string().required(
-        textKeys.GENERIC_ERROR_INPUT_REQUIRED(),
-      ),
+      registrationNumber: Yup.string()
+        .matches(/^[A-Z]{3}\d{3}$/, textKeys.GENERIC_ERROR_INPUT_FORMAT())
```

Comment:
There is a reg number regex already that you should re-use: `SE_CAR_REGISTRATION_NUMBER_REGEX`